core/data.go

```diff
@@ -171,3 +171,17 @@ func (cb Bundles) Size() int64 {
 	}
 	return size
 }
+
+// Sample is a chunk with associated metadata used by the Universal Batch Verifier
+type Sample struct {
+	Commitment      *Commitment
+	Chunk           *Chunk
+	AssignmentIndex ChunkNumber
+	BlobIndex       int
+}
+
+// SubBatch is a part of the whole Batch with identical Encoding Parameters, i.e. (ChunkLen, NumChunk)
```

Comment:
Can you comment if all blobs with same encoding params will be in the same SubBatch?